src/test/rustdoc-gui/item-info-alignment.goml

```diff
@@ -0,0 +1,11 @@
+// This test ensures that the "item-info" looks about the same
+// whether or not it's inside a toggle.
+goto: file://|DOC_PATH|/lib2/struct.ItemInfoAlignmentTest.html
+
+// First, we try it in "desktop" mode.
+size: (1200, 870)
+compare-elements-property: (".impl-items > .item-info", "summary > .item-info", ["scrollLeft"])
```

Comment:
Instead of this command, maybe `compare-elements-position` might be better?

```suggestion
compare-elements-position: (".impl-items > .item-info", "summary > .item-info", ("x"))
```